### PR TITLE
BaseTools/Scripts: Remove Cc: tag check from PatchCheck.py

### PR DESCRIPTION
**Pull request by mdkinney**
The commit message format requirements have been updated for GitHub PR based code reviews and no longer required Cc: tags for the maintainers and reviewers.  Remove the Cc: tag check from PatchCheck.py.

### CODE REVIEW

**Review by makubacki — APPROVED** [review]

**Review by bcran — APPROVED** [review]

**Comment by bcran** [comment]
I see merging is blocked until someone with write access reviews it.
Who has write access with the new process?

**Comment by mdkinney** [comment]
The settings during the soft/hard freeze were not reset yet.  I have updated those for EDK II Maintainers with Write permissions.

Reviews completed by edk2 maintainers of BaseTools.

**Pull request merged**
